README.md

```diff
@@ -1,8 +1,14 @@
 # Vouch Register for Solver Bonding
 
-The provided contract allows bonding pools - as described in this [cip](https://snapshot.org/#/cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7) - to either vouch or invalidate a vouch for a solver.
-The provided contract must be called by addresses that created the bonding pool. Calls from other addresses will not be officially indexed and it will not start the allowlisting process for new solvers for the COW-Protocol.
-In case of conflicting vouching votes (e.g. two bonding pools vote for the same solver with different cow reward targets, the first vouch will be seen as the valid one).
+A _bonding pool_ is a Gnosis Safe instance that is owned solely by the CoW DAO.
+Its creator escrows the amount specified in the [CIP](https://snapshot.org/#/cow.eth/proposal/0x267edf7a0bd3c771cfca763322f011ee106d8d5158612c11da29183260d1dba7) to guarantee that none of the vouched solvers misbehaves.
+The provided contract allows bonding pools to either vouch or invalidate a vouch for a solver.
+
+The provided contract must be called by the owner of the bonding pools. The owner of a bonding pool is defined as the address that escrowed the initial funding to the bonding pools - in case there are several addresses sending the initial funds to the same bonding pool, the bonding pool can not be considered.
+Calls from other addresses will not be officially indexed and it will not start the allowlisting process for new solvers for the COW-Protocol.
```

Comment:
This means then that we need to hard code the expected `sender` addresses in our queries in order to filter out the noise from this contract. Right?

Reply:
In dune, we could also just hardcode the bonding pools and then index the owners of the bonding pool by investigating the addresses from which the funds were sent.